src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,93 @@
+{% for resource in collections['resources_' + lang] %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("projects", resource.data.projects) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("types", resource.data.type) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("topics", resource.data.topics) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("publishers", resource.data.publisher) %}
+{% endfor %}
+<div id="filters" class="filters flow">
+    <h3>{% __ 'filters' %}</h3>
+    <div class="flow">
+        {% if collections['topics_' + lang] | length > 1 %}
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'topic' %}</legend>
+            {% disclosure 'topic' %}
+            <ul id="filter-topic">
+                {% for topic in collections['topics_' + lang] %}
+                    {% if topic.data.uuid in filterableOptions.topics %}
+                    <li>
+                        <input type="checkbox" id="{{ topic.data.uuid }}" name="topic" value="{{ topic.data.uuid }}" data-filter-key="topics" />
+                        <label for="{{ topic.data.uuid }}">{{ topic.data.title }}</label>
+                    </li>
+                    {% endif %}
+                {% endfor %}
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
+        {% endif %}
+        {% if filterableOptions.types | length > 1 %}
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'type' %}</legend>
+            {% disclosure 'type' %}
+            <ul id="filter-type">
+                {% for value in filterableOptions.types %}
+                <li>
+                    <input type="checkbox" id="{{ value }}" name="type" value="{{ value }}" data-filter-key="type" />
+                    <label for="{{ value }}">{% __ value %}</label>
+                </li>
+                {% endfor %}
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
+        {% endif %}
+        {% if filterableOptions.projects | length > 1 %}
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'project' %}</legend>
+            {% disclosure 'project' %}
+            <ul id="filter-project">
+                {% for project in collections['projects_' + lang] %}
+                    {% if project.data.uuid in filterableOptions.projects %}
+                    <li>
+                        <input type="checkbox" id="{{ project.data.uuid }}" name="project" value="{{ project.data.uuid }}" data-filter-key="projects" />
+                        <label for="{{ project.data.uuid }}">{{ project.data.title }}</label>
+                    </li>
+                    {% endif %}
+                {% endfor %}
+            </ul>
+            {% enddisclosure %}
+        </fieldset>
+        {% endif %}
+        {% if collections['resources_fr'] | length > 0 %}
```

Comment:
This isn't quite right. The French resources collection is not a collection of resources in French; it's a translation of the resource information so even French-speaking visitors can view the index of resources. So really you should be compiling `filterableOptions.language` based on the `language` data for each resource.